docs/hub/oauth.md

```diff
@@ -35,6 +35,7 @@ The currently supported scopes are:
 - `write-repos`: Get write/read access to the user's personal repos.
 - `manage-repos`: Get full access to the user's personal repos. Also grants repo creation and deletion.
 - `inference-api`: Get access to the [Inference API](https://huggingface.co/docs/api-inference/index), you will be able to make inference requests on behalf of the user.
+- `write-discussions`: Open discussions and pull requests on behalf of the user, and interact with discussions.
```

Comment:
What does interact mean? Reactions? Replying to?

Reply:
yes, reactions, comments, editing comments, closing discussions, changing the title, ...

Reply:
add this between parenthesis?